src/main/java/Controller/LoginController.java

```diff
@@ -0,0 +1,26 @@
+package Controller;
+
+import InputBoundary.loginInputBoundary;
+import InputData.loginInputData;
+
+
+/**
+ * Controller for login and register use cases.
+ */
+public class LoginController {
+    loginInputBoundary loginInteractor;
+    /**
+     * Constructor.
+     * @param loginInteractor use case interactor for user logging in
+     */
+    public LoginController(loginInputBoundary loginInteractor) {
```

Comment:
As we discussed just earlier today, please don't pass the interactor into the controller; the controller should construct the interactor and the input data.